jetty-util/src/main/java/org/eclipse/jetty/util/Callback.java

```diff
@@ -39,6 +39,24 @@ public InvocationType getInvocationType()
         }
     };
 
+    /**
+     * <p>Completes this callback with the given {@link CompletableFuture}.</p>
+     * <p>When the CompletableFuture completes normally, this callback is succeeded;
+     * when the CompletableFuture completes exceptionally, this callback is failed.</p>
+     *
+     * @param completable the CompletableFuture that completes this callback
+     */
+    default void completeWith(CompletableFuture<?> completable)
```

Comment:
Should this be `CompletableFuture<Void>`?

Reply:
No, because sometimes we pass in a `CompletableFuture<Stream>` as we don't care about the result, but just to complete the `Callback`, so the wildcard is necessary.